3_Risk.md

```diff
@@ -1,37 +1,47 @@
 # Metric: Risk
 
+Goal: Identify the risk profile of a project. 
+
 ## Human Factors
-Activity Metric | Description
+
+Goal: Identify the risk associated with the people involved in a project. 
+
+Metric | Question
 --- | ---
-Contributor Importance | Percentage of commits by individual contributors from identified organizations over time.
-Qualified Committers | Contributions over time and what components they commit to over time.
-User Dependency | Number of users who are aware that they depend on the software over time.
-Paid Developers | Number of paid developers in community over time.
+Contributor Importance | What is the percentage of commits by individual contributors from identified organizations?

```

Comment:
Maybe we could express it in terms of "fraction" instead of "percentage"? Even if we later use percentage as the kind of fraction....

Reply:
@germonprez made the suggested change. Thanks for the suggestion @jgbarah .